README.Rmd

```diff
@@ -184,79 +176,23 @@ pbmc_small_polished %>%
     geom_boxplot(outlier.shape=NA) +
     geom_jitter(aes(size=nCount_RNA), alpha=0.5, width=0.2) +
     scale_y_log10() +
-    my_theme
-```
-
-# Quality control
-
-We can quality control for each sample in an intuitive way using nest function and Bioconductor utilities
-
-```{r}
-
-location <- mapIds(
-	EnsDb.Hsapiens.v86, 
-	keys=rownames(pbmc_small_polished), 
-	column="SEQNAME",
-	keytype="SYMBOL"
-)
-
-counts_mito = 
-	pbmc_small_polished %>%
-	
-	nest(data = -sample) %>%
-	mutate(data = map(
-		data, 
-		~ .x %>%
-			
-			# Join mitochondrion statistics
```

Comment:
I am a bit confused if these changes fix the figure, how come there is a bunch of code that was deleted?

Reply:
So when I tried to render the README.Rmd to update the README.md it kept erroring (e.g. with EnsDb.Hsapiens.v86) and I saw that there was code in it that wasn't in the Introduction vignette. The README looked like it was older than the Introduction vignette so I replaced it with the Introduction code. The README code is now an exact copy of what's in the Introduction Rmd, should it be?

Reply:
> So when I tried to render the README.Rmd to update the README.md it kept erroring (e.g. with EnsDb.Hsapiens.v86) and I saw that there was code in it that wasn't in the Introduction vignette. The README looked like it was older than the Introduction vignette so I replaced it with the Introduction code. The README code is now an exact copy of what's in the Introduction Rmd, should it be?

Amazing!